package.json

```diff
@@ -32,7 +32,7 @@
     "fs-extra": "^8.1.0",
     "html-postcss": "^0.1.2",
     "merge-stream": "^2.0.0",
-    "polymer-build": "^3.1.2",
+    "polymer-build": "git+https://github.com/BBVAEngineering/polymer-build.git#v3.2.0",
```

Comment:
Can we use a published package instead of a repo tag?

Reply:
We can use a published package changing the package name or using a scope...

Reply:
Did you open a PR to the main package in order to try to merge the change?

Reply:
I can try it but it's not going to be merged. There are more than 1000 open issues and the repo is unmaintained. I think that this package (polymer-build) is not going to receive new PRs and wont require maintenance. However, if we publish it with another name, we have to maintain it, set a proper publish / build process, etc.

Another option, not good (very bad in fact) since this package requires modules of polymer-build explicitly, is to include it as peerDependency and let the app specify the version to use. 

Third option: include the package as a bundledDependency.

Fourth: move everything (polymer-build, this package and ember-cli-polymer-bundler) to private repos. I think that this option does not deserve the effort for the time we, and maybe other people, are going to use Polymer.

As a last resource, I can publish it in npm from my Github account. I'm not going to maintain the repo, except for security updates, but at least it will be published in npm.

Reply:
PR in Polymer/tools: https://github.com/Polymer/tools/pull/3502

Reply:
Changed to build-polymer (published in npm) instead of polymer-build.
I don't have permissions to delete the fork of polymer-build in BBVAEngineering.

Reply:
If a leave the company and the team needs to make some modification to that package, you'll probably need to make a fork again... That was the point of moving the fork here.